authz/providers/azure/azure.go

```diff
@@ -94,27 +94,25 @@ func (s Authorizer) Check(request *authzv1beta1.SubjectAccessReviewSpec, store a
 	if _, ok := request.Extra["oid"]; !ok {
 		if s.rbacClient.ShouldSkipAuthzCheckForNonAADUsers() {
 			glog.V(3).Infof("Skip RBAC is set for non AAD users. Returning no opinion for user %s. You may observe this for AAD users for 'can-i' requests.", request.User)
```

Comment:
should this be 3? or above 5? it's very verbose and provides no value. remember people enabling guard log will pay money base on volume

Reply:
Rational to keep this was, for cert user or AAD user using impersonation, we may deny the request if they don't set oid value in kubeconfig however verdict msg will inform user hence can set it as 7 now.

Reply:
it will get printed in all SA request in AKS

Reply:
i think you already have the specific reason to return, why log here again?

Reply:
This will print all cert users and AAD users who are trying impersonate without giving oid of impersonated user. As we have moved response to level 7, we won't have any idea what we return for these scenarios in normal trace level. Although error message is self explanatory we can update document and remove this log line.

Reply:
removed the log line